CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to the Sorare GraphQL API will be documented in this file. W
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## 2023-05-03
+
+Removed the following deprecated field:
+
+- `Card.priceRange`
```

Comment:
Can you describe what to use instead here?

Reply:
AFAICS we describe the alternative when we deprecate but not when we remove

Reply:
@sfriquet @redox Can you help explain the exact meaning of token{priceRange{min max}} ? How is it computed ?
 Also, observed values are surprising. Ex for diego-jose-palacios-espinoza:
               "priceRange": {
                "min": "6855031737193763",
                "max": "68550317371937638"
              }